crates/aranya-daemon/src/policy.md

```diff
@@ -481,16 +576,136 @@ command TerminateTeam {
 - Only an Owner can create this event.
 - Once terminated, no further communication will occur over the team graph.
 
+## SetupDefaultRoles
+
+The `SetupDefaultRoles` command sets up default roles on the team.
+Operation can only be invoked by the team owner role.
+
+```policy
+// Setup default roles on a team.
+action setup_default_roles() {
+    publish SetupAdminRole {}
+    publish SetupOperatorRole {}
+    publish SetupMemberRole {}
+}
+
+command SetupAdminRole {
+    fields {}
+
+    seal { return seal_command(serialize(this)) }
+    open { return deserialize(open_envelope(envelope)) }
+
+    policy {
+        // Get author of command
+        let author = get_valid_device(envelope::author_id(envelope))
+
+        check device_can_execute_op(author.device_id, "SetupAdminRole")
+
+        // A role's ID is the ID of the command that created it.
+        let role_id = envelope::command_id(envelope)
+
+        let role = RoleInfo {
+            role_id: role_id,
+            name: "admin",
+            author_id: author.device_id,
+        }
+
+        finish {
+            create_role_fact(role)
+            assign_op_role("DeleteLabel", role.role_id)
+
+            emit QueriedRole {
```

Comment:
You want `RoleCreated`, not `QueriedRole`. Same for the other two default roles.